sass/layout/_index.scss

```
@@ -3,49 +3,35 @@
 /// - Organize named z-index values in a list
 /// - Manage all breakpoints in a map,
 ///   and access with media-query helpers
-/// - Use our positioning, clearfix, and box-model utilities
 /// @group layout
 
 
-@import 'box-sizing';
-@import 'clearfix';
-@import 'position';
-@import 'query-utils';
-@import 'queries';
-@import 'ratio';
-@import 'z-index';
+@forward 'query-utils' hide nudge-query;
+@forward 'queries';
+@forward 'z-index';
 
 
 /// ## Import
-/// If you already imported `accoutrement/sass/tools` you are ready to go.
-/// If you want to import individual plugins,
-/// make sure you first import the shared core:
 ///
-/// ```scss
-///   @import '<path-to>/accoutrement/sass/core';
-///   @import '<path-to>/accoutrement/sass/plugin/layout';
-/// ```
-///
-/// If you're using [Eyeglass](https://github.com/linkedin/eyeglass),
-/// you can leave off most of the path:
+/// If you already imported `accoutrement/sass/tools` you are ready to go.
+/// You can also import the layout module on its own:
 ///
 /// ```scss
-///   @import 'accoutrement/core';
-///   @import 'accoutrement/plugin/layout';
+///   @use '<path-to>/accoutrement/sass/layout';
 /// ```
 /// @group layout
 
 
 /// ## Media Queries
 /// Establish your media-query breakpoints,
-/// or use sizes from [Accoutrement Scale][scale.html] directly:
+/// or use sizes from [Accoutrement Scale](scale.html) directly:
 ///
 /// ```scss
-/// $sizes: (
+/// tools.$sizes: (
```

```suggestion
/// $sizes: (
```

We haven't been adding namespaces in markdown prose comments - only in sassdoc `@example`s.